mk/config.mk

```diff
@@ -356,6 +356,9 @@ $(eval $(call cfg-depends-all,CFG_SECSTOR_TA_MGMT_PTA,CFG_SECSTOR_TA))
 # GlobalPlatform Core API (for example, re-seeding RNG entropy pool etc.)
 CFG_SYSTEM_PTA ?= y
 
+# Enable the pseudo TA for enumeration of TEE based devices for Linux
+CFG_DEVICE_PTA ?= y
```

Comment:
Maybe prefer `CFG_REE_DEVICE_PTA` if this is dedicated to PTA reachable from the REE.

I found also `TA_FLAG_KERNEL_DEVICE` not very obvious, Kernel device here relates to OP-TEE core or REE kernel device drivers? 
Since this is about discoverablility of the service, maybe `TA_FLAG_ENUMERATED` is more appropriate?

Reply:
As @jforissier suggested, I think `CFG_DEVICE_ENUM_PTA` looks more appropriate. Also accordingly for flag: `TA_FLAG_DEVICE_ENUM`.